src/main/resources/application-production.properties

```diff
@@ -0,0 +1,38 @@
+spring.mail.host=${SPRING_MAIL_HOST}
+spring.mail.port=${SPRING_MAIL_PORT}
+payments.mail.sender=${PAYMENTS_MAIL_SENDER}
+
+#Thymeleaf
+spring.thymeleaf.check-template-location=true
+spring.thymeleaf.prefix=classpath:/templates/
+spring.thymeleaf.suffix=.html
+spring.thymeleaf.mode=HTML5
+spring.thymeleaf.encoding=UTF-8
+spring.thymeleaf.content-type=text/html
+
+##DATABASE PROPERTIES
+spring.datasource.driverClassName=org.hsqldb.jdbc.JDBCDriver
+spring.datasource.url=${SPRING_DATASOURCE_URL}
+spring.datasource.username=${SPRING_DATASOURCE_USERNAME}
+spring.datasource.password=${SPRING_DATASOURCE_PASSWORD}
+spring.data.jpa.repositories.enabled=true
+spring.jpa.show-sql=false
+spring.jpa.database-platform=org.hibernate.dialect.HSQLDialect
+spring.jpa.database=HSQL
+spring.jpa.generate-ddl=true
+spring.jpa.hibernate.ddl-auto=validate
+spring.jpa.properties.hibernate.default_schema=PUBLIC
+endpoints.cors.allow-credentials=*
+endpoints.cors.allowed-origins=*
+endpoints.cors.allowed-methods=*
+endpoints.cors.allowed-headers=*
+endpoints.cors.max-age=3600
+security.basic.enabled=false
+payments.paymentReturnUrl=http://ch.tudelft.nl
```

Comment:
I think we do, since this is the fallbackUrl if no returnUrl is specified for a payment. We'll have to check all url functions though, since it's not quite clear what url is supposed to do what in the properties, it could definitely use some more clarity.